packages/fx-core/src/question/create.ts

```diff
@@ -1452,7 +1452,9 @@ export function getLanguageOptions(inputs: Inputs): OptionItem[] {
     // SPFx only supports typescript
     return [{ id: ProgrammingLanguage.TS, label: "TypeScript" }];
   } else if (
-    capabilitiesHavePythonOption.includes(capabilities) &&
+    capabilitiesHavePythonOption.includes(
+      inputs[capabilities] ? inputs[capabilities] : capabilities
```

Comment:
why we need to add check for `input[capabilities]` here?

Reply:
for custom-copilot-rag-customize and custom-copilot-rag-azureAISearch, there capabilities are both `custom-copilot-rag` but `inputs[custom-copilot-rag]` are different.
I want to make the judgement more specific on app type not capability type.